mix.exs

```diff
@@ -38,7 +38,8 @@ defmodule Metatags.Mixfile do
 
   defp deps do
     [
-      {:phoenix_html, "~> 2.9"},
+      {:phoenix_html, "~> 3.1"},
+      {:plug, "~> 1.12.1"},
```

Comment:
Maybe we could be a bit looser with the plug version just in case someone is using an older version and to ensure further minor updates causes dependency clashes.

```suggestion
      {:plug, "~> 1.0"},
```